static/app/components/forms/selectOption.tsx

```diff
@@ -0,0 +1,145 @@
+import {components as selectComponents} from 'react-select';
+import styled from '@emotion/styled';
+
+import {IconCheckmark} from 'sentry/icons';
+import space from 'sentry/styles/space';
+
+type Props = React.ComponentProps<typeof selectComponents.Option>;
+
+const Option = (props: Props) => {
+  return (
+    <Wrap>
+      <selectComponents.Option {...props}>
+        <InnerWrap isFocused={props.isFocused}>
+          <Indent>
+            <CheckWrap isMulti={props.isMulti} isSelected={props.isSelected}>
+              {props.isSelected && (
+                <IconCheckmark
+                  size={props.isMulti ? 'xs' : 'sm'}
+                  color={props.isMulti ? 'white' : undefined}
+                />
+              )}
+            </CheckWrap>
+            {props.data.leadingItems && (
+              <LeadingItems>{props.data.leadingItems}</LeadingItems>
+            )}
+          </Indent>
+          <ContentWrap
+            isFocused={props.isFocused}
+            showDividers={props.selectProps.showDividers}
+          >
+            <Text>
+              <Label>{props.label}</Label>
+              {props.data.details && <Details>{props.data.details}</Details>}
+            </Text>
+            {props.data.leadingItems && (
+              <TrailingItems>{props.data.trailingItems}</TrailingItems>
+            )}
+          </ContentWrap>
+        </InnerWrap>
+      </selectComponents.Option>
+    </Wrap>
+  );
+};
+
+export default Option;
+
+const Wrap = styled('div')``;
```

Comment:
Do we even need this as a styled component?

Reply:
Only added it so I can do
```
${Wrap}:last-of-type & {
    box-shadow: none;
}
```
down below in `ContentWrap`. Point is to remove the bottom border (implemented as box-shadows) in bottom options.

Reply:
ah yea, cool.